app/views/exclusion/ExclusionNinoOrNoNinoForm.scala.html

```diff
@@ -39,67 +39,76 @@
 
     @defining( if(isCurrentTaxYear=="cy"){""+taxYearRange.cyminus1}else{""+taxYearRange.cy} ) { yearvalue =>
         @back_link(gaLabel = messages("ExclusionNinoDecision.title"))
-        <header class="page-header">
-            <h1 id="title">
-                @messages("ExclusionNinoDecision.title")
-            </h1>
-        </header>
 
-        <section id="exclusion-decision">
-            @if(flash.get("error")){
-                <div class="error-summary-pbik" role="group" aria-labelledby="error-summary-heading-1" tabindex="-1">
+        @if(flash.get("error")){
+            <div class="error-summary" role="group" aria-labelledby="error-summary-heading" tabindex="-1">
```

Comment:
Are we able to replace this bit of html with the default errorSummary helper? (I mean the lines below what I've highlighted here too)